crml/sylo/src/groups/default_weights.rs

```diff
@@ -56,7 +56,9 @@ impl crate::groups::WeightInfo for () {
 	fn accept_invite() -> Weight {
 		0
 	}
-	fn revoke_invites() -> Weight {
-		0
+	fn revoke_invites(_i: usize) -> Weight {
```

Comment:
was it intentional to exclude `_i` here?

Reply:
Yes, in theory, the length of the revoked list should affect the performance. In practical and within our limits (the max invites and etc), it didn't depend on it. However, as the benchmark remains parametric, we should keep this.